recipes/fastgltf/all/conanfile.py

```diff
@@ -67,7 +67,11 @@ def layout(self):
         cmake_layout(self, src_folder='src')
 
     def requirements(self):
-        self.requires("simdjson/3.2.0")
+        # See https://github.com/spnda/fastgltf/commit/aa90dfac290584f35f47dd7c1634f23d376a7207
+        if Version(self.version) >= "0.6.0":
+            self.requires("simdjson/3.7.0")
+        else:
+            self.requires("simdjson/3.2.1")
```

Comment:
```suggestion
        self.requires("simdjson/3.7.0")
```
Are the older versions incompatible with the latest simdjson?

Reply:
Yes. I originally had what you are suggesting, but removed that since it caused build failures on older versions.

Version 3.2.2 moved around a lot of headers and includes, which broke my base64 decoders as they didn't include the headers properly, and the code only worked because `simdjson.h` leaked the SIMD intrinsics. I fixed that in 0.6.0, which is why all versions since then have supported any of the more recent versions of simdjson. See https://github.com/simdjson/simdjson/pull/2039.

Reply:
As noted in https://github.com/conan-io/conan-center-index/pull/22960#issuecomment-2294509192, I would be happy to remove these legacy versions of fastgltf since they're pretty old and include a large number of issues. And I don't think anybody actually uses them anymore.